city_scrapers/spiders/chi_southwest_home_equity_i.py

```diff
@@ -0,0 +1,173 @@
+        )
+
+        for agenda_node in table.xpath(
+            "div[@data-ux=\"GridCell\"][contains(., 'Agenda')]"
+        ):
+            minutes_node = agenda_node.xpath(
+                "following-sibling::div[@data-ux=\"GridCell\"][contains(., 'Minutes')]"
+            )
+
+            if self._get_link(minutes_node):
+                minutes_contents = self.parse_pdf(self._get_link(minutes_node))
+            else:
+                minutes_contents = None
+
+            meeting = Meeting(
+                title=(
+                    self._parse_title(minutes_contents)
+                    if minutes_contents
+                    else "Board Meeting"
+                ),
+                description="",
+                classification=COMMISSION,
+                start=self._parse_start(agenda_node),
+                end=None,
+                all_day=False,
+                time_notes=(
+                    self._parse_time_notes(minutes_contents)
+                    if minutes_contents
+                    else None
+                ),
+                location=self.location,
+                links=self._parse_links([agenda_node, minutes_node]),
+                source=self._parse_source(response),
+            )
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+            yield meeting
+
+    def _parse_title(self, text):
+        name = re.search(r"Southwest Home Equity Assurance Program", text, flags=re.I)
+        minutes = re.search(r"MINUTES", text, flags=re.I)
+        if name and minutes:
+            title = text[name.end() : minutes.start()]
+            return title.replace("\n", "").title().strip()
+        else:
+            return "Board Meeting"
+
+    def parse_pdf(self, url):
+        response = requests.get(url)
+        pdf_text = extract_text(BytesIO(response.content))
+        return pdf_text
+
+    def _parse_start(self, node):
+        # Try to get date from title
+        try:
+            date_str = self.find_date_in_title(node)
+            date_object = datetime.strptime(date_str, "%B %d %Y")
+        except (RuntimeError, ValueError):
+            try:
+                date_str = self.find_date_in_content(node)
+                date_object = datetime.strptime(date_str, "%B %d %Y")
+            except (RuntimeError, ValueError):
+                return None
+
+        try:
+            start_time = self.find_time(node)
+            return date_object + start_time
+
+        except (RuntimeError, ValueError):
+            return date_object + timedelta(hours=18, minutes=30)
+
+    def _parse_links(self, nodes):
+        links = []
+        for node in nodes:
+            if node:
+                links.append(
+                    {"title": self._get_name(node), "href": self._get_link(node)}
+                )
+        return links
+
+    def _get_name(self, node):
+        name = node.xpath("string(.)").get()
```

Comment:
Since we know that the overwhelming majority of docs will be either Agenda or Minutes, we can check if either of those are present in the string and return "Agenda" or "Minutes" statically if so to keep things clean.